File: Alimento.java
public class Alimento extends Producto {
    private String caducidad;

    public Alimento(String nombre, double precio, String fechaImportacion, int stock, String caducidad) {
        super(nombre, precio, fechaImportacion, stock);
        this.caducidad = caducidad;
    }

    public String obtenerInformacionConCaducidad() {
        return super.obtenerInformacion() + " Caducidad: " + caducidad;
    }
}

File: Alimento.py
from Producto import Producto

class Alimento(Producto):
    def __init__(self, nombre, precio, fecha_importacion, stock, caducidad):
        super().__init__(nombre, precio, fecha_importacion, stock)
        self.caducidad = caducidad

    def obtener_informacion_con_caducidad(self):
        return super().obtener_informacion() + " Caducidad: " + self.caducidad

    def get_nombre_producto(self):
        return super().get_nombre()
    
    
